features/storage/blockdevice/ObservingBlockDevice.cpp

```diff
@@ -36,7 +35,7 @@ ObservingBlockDevice::~ObservingBlockDevice()
     // Does nothing
 }
 
-void ObservingBlockDevice::attach(Callback<void(BlockDevice *)> cb)
+void ObservingBlockDevice::attach(mbed::Callback<void(BlockDevice *)> cb)
```

Comment:
why not adding using namespace mbed?

Reply:
Consistency, also after this cleanup next PR is to add all block devices and filesystem into the mbed namespace

Reply:
I can live with it. Specifically if next PR is to bring those block devices into mbed namespace.

Reply:
I have that PR already in place, but cannot merge unless this cleanup is done